src/components/Modal/index.jsx

```diff
@@ -0,0 +1,28 @@
+import { useState } from "react";
+
+import ProfilePage from "../../pages/ProfilePage";
+
+import { DefaultButton } from "../buttons/style";
+import { Title, Form, ProfileImage, Input, NewProfileContainer } from "./style";
+
+export const SelectProfileModal = () => {
+  const [isFormSubmit, setFormSubmit] = useState(null);
+
+  const handleFormSubmit = (e) => {
+    setFormSubmit(e.target.value)
+  }
+  return (
+    <>
+      <Form style={{justifyContent: 'left'}}>
+        <Title>Add New Profile</Title>
+
+        <div>
```

Comment:
Is this a regular div? Maybe it would be good to have a styled compoment container here.

Reply:
The purpose of this div would be to let the both profileImage and Input at the same level (like a container?) and giving them the proper style. I thought to add it to a styled but it's actually doing nothing (if you check the line 20 of style.jsx there's an empty component that I must delete

Reply:
If there's no styles to add to it, then i guess there's no need for a styled component